tests/python/gaia-ui-tests/gaiatest/apps/system/regions/search_panel.py

```diff
@@ -0,0 +1,109 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import urllib
+
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+from gaiatest.apps.base import PageRegion
+
+
+class SearchPanel(Base):
```

Comment:
I just copied this file from the homescreen folder, I think we have quite a few more of these files under regions, that use the `Base` page class.
I can add a comment that it is temporary, but should I mention a bug comment or something?